libqtile/backend/x11/core.py

```diff
@@ -163,7 +162,7 @@ def get_screen_info(self) -> List[Tuple[int, int, int, int]]:
         # important, because it indicates what people have chosen via xrandr
         # --primary or whatever. So we need to alias screens that should be
         # aliased, but preserve order as well. See #383.
-        xywh = OrderedDict()  # type: Dict[Tuple[int, int], Tuple[int, int]]
+        xywh = {}
```

Comment:
seems like we're losing the type hint here? rest of it looks good to me

Reply:
yeah, didn't think we needed it anymore(we don't with pytype). We still do!

Reply:
We don't with pytype? not sure I understand what you mean there. if we really can drop it, that's fine.

Reply:
mypy requires this type hint here. pytype does not, since it does type inference.

It's still needed as long as we use mypy, I've just been tinkering with getting pytype working on my local(it provides better hints)

Reply:
great, thanks for that. pytype would be great if it can do more of this for us. maybe i should look into it for qtile-check too...